docs/spec/spec.md

```diff
@@ -1652,14 +1652,15 @@ The `graphiql` field is used to provide the GraphiQL client configuration to ena
 @graphql:ServiceConfig {
     graphiql: {
         enabled: true,
-        path: "/ballerina/graphiql"
+        path: "/ballerina/graphiql",
+        printUrl: false
     }
 }
 service on new graphql:Listener(9090) {
     // ...
 }
 ```
->**Note:** The field enabled accepts a `boolean` that denotes whether the client is enabled or not. By default, it has been set to `false`. The optional field `path` accepts a valid `string` for the GraphiQL service. If the path is not given in the configuration, `/graphiql` is set as the default path.
+>**Note:** The field `enabled` accepts a `boolean` that denotes whether the client is enabled or not. By default, it has been set to `false`. The optional field `path` accepts a valid `string` for the GraphiQL service. If the path is not given in the configuration, `/graphiql` is set as the default path. The optional field `printUrl` accepts a boolean that denotes whether the GraphiQL url is printed to stdout or not. By default, it has been set to `true`.
```

Comment:
LEt's break down the fields into separate subtitles.

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-graphql/pull/1524/commits/e5f1c907452289fa565cd7d2cdd0d1bb1cc2c776